tests/unit/s2n_security_policy_cert_preferences_test.c

```diff
@@ -15,25 +15,84 @@
 
 #include "s2n_test.h"
 #include "testlib/s2n_testlib.h"
+#include "tls/s2n_certificate_keys.h"
 #include "tls/s2n_security_policies.h"
 #include "tls/s2n_signature_scheme.h"
 
+#define CHAIN_LENGTH 3
+
+static S2N_RESULT s2n_test_construct_cert_chain(
+        struct s2n_cert *certs,
```

Comment:
Nit: I know this is just a test, but any array with an assumed size makes me uneasy ;) Any reason not to pass the size in explicitly?